### PR TITLE
(8151): Updated vpn user resource to support saml_endpoint refresh

### PR DESCRIPTION
**Pull request by zhiqiangjin**

### CODE REVIEW

File: aviatrix/resource_vpn_user.go [hunk 1]
```diff
@@ -92,9 +94,7 @@ func resourceAviatrixVPNUserRead(d *schema.ResourceData, meta interface{}) error
 		if vu.UserEmail != "" {
```
Comment (arvindsree) [inline aviatrix/resource_vpn_user.go]: Don't need empty check here as well

File: aviatrix/resource_vpn_user.go [hunk 2]
```diff
@@ -60,7 +61,8 @@ func resourceAviatrixVPNUserCreate(d *schema.ResourceData, meta interface{}) err
 		return fmt.Errorf("failed to create Aviatrix VPNUser: %s", err)
 	}
 	d.SetId(vpn_user.UserName)
-	return nil
+	//return nil
+	return resourceAviatrixSpokeVpcRead(d, meta)
```
Comment (arvindsree) [inline aviatrix/resource_vpn_user.go]: Spoke VPC read for VPN user????

File: aviatrix/resource_vpn_user.go (hunk 1 above)
Comment (zhiqiangjin) [reply aviatrix/resource_vpn_user.go]: Thanks for pointing it out.

File: aviatrix/resource_vpn_user.go (hunk 2 above)
Comment (zhiqiangjin) [reply aviatrix/resource_vpn_user.go]: Thanks for pointing it out. Have corrected it.

**Pull request merged**